Decompiler/PscDecompiler.hpp

```diff
@@ -11,7 +11,7 @@
 
 #include "Node/Base.hpp"
 
-//#define TRACE_DECOMPILATION
+#define TRACE_DECOMPILATION
```

Comment:
Trace should likely be off by default, possibly shifting it to a runtime flag in the future